virttest/utils_test/__init__.py

```diff
@@ -698,6 +698,27 @@ def run_autotest(vm, session, control_path, timeout,
     from autotest.client.shared.settings import settings
     section_values = settings.get_section_values
 
+    def install_client_tests(url, autotest_path, timeout=600):
```

Comment:
Well the only issue I have with this is it's installing the `client` tests on the host only to copy it to guest. How about executing this only in the VM after we copy the autotest?